app/store/aliases.ts

```diff
@@ -29,7 +29,9 @@ const fetchDataAlias = (payload: payload) => {
       requestHeadersObject
     })
       .then((data: axios.AxiosResponse) => {
-        const stringifiedData = JSON.stringify(data.data);
+        const stringifiedData = data.headers["content-type"].includes("json")
```

Comment:
Why we need this condition?

Reply:
we dont need the pretty printing if the `content-type` is anything other than `application/json`. I had earlier added a `strict` equals condition for checking, but turns out that some websites sent out `content-type` headers as `application/json; charset=utf-8`. So this condition is needed to filter out types like `text/html`/`text/plain` or `application/x-www-form-urlencoded` for instance

Reply:
okey. but in the false part of this condition we are doing `JSON.stringify` doesn't that break for `text/html`?

Reply:
This is quite strange. I logged out the following 

![bug-json](https://user-images.githubusercontent.com/5921327/42273118-e50fc48c-7fa5-11e8-879d-2a74843177d0.png)

and intercepted a svg file, for which the output is:

![bug--json](https://user-images.githubusercontent.com/5921327/42273164-0901e118-7fa6-11e8-922b-38bef44a0ab5.png)

running the following on chrome `console` or `node repl` gives me the following error

![chrome-console](https://user-images.githubusercontent.com/5921327/42273292-6947daaa-7fa6-11e8-8c71-0836605d255e.png)

but we are getting the proper output for `stringified data` as you can see in first screenshot instead of it throwing an error. Not sure what is happening here.

Reply:
This is what happens if I don't stringify the data for `svg` type (and what should happen ideally)

![screen shot 2018-07-04 at 4 30 25 pm](https://user-images.githubusercontent.com/5921327/42273760-c3c0e318-7fa7-11e8-83a8-8ac43318b8f5.png)

Reply:
I mean without `JSON.stringify` does the correct data coming to `textbox`?
like without escape?

Reply:
yes, for non-json types, correct data is coming through. for `json` type, i am getting `[object Object]`

Reply:
then why can't we remove the `JSON.stringify` for non JSON-types.

Reply:
Done. Please check